sources/en/r/raeitranslations.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import logging
+from urllib.parse import urlencode
+
+from lncrawl.core.crawler import Crawler
+from lncrawl.models import Chapter
+
+logger = logging.getLogger(__name__)
+
+
+class MyCrawlerName(Crawler):
+    base_url = ["https://raeitranslations.com/"]
+    novel_json_url_prefix = "https://api.raeitranslations.com/api/novels/"
+    novel_cover_url_prefix = "https://raeitranslations.com/assets/"
+    novel_chapters_list_url_prefix = "https://api.raeitranslations.com/api/chapters/list?"
+    novel_chapter_url_prefix = "https://api.raeitranslations.com/api/chapters/single?"
+    json_request_headers = {
+        "Accept": "*/*",
+        "Origin": "https://raeitranslations.com",
+        "Referer": "https://raeitranslations.com/"
+    }
+
+    def read_novel_info(self) -> None:
+        title = self.novel_url.rstrip("/").split("/")[-1]
+        novel_json = self.get_json(
+            f"{self.novel_json_url_prefix}{title}",
+            headers=self.json_request_headers
```

Comment:
headers are already added inside scraper. do you need to provide them separately?
check: https://github.com/dipu-bd/lightnovel-crawler/blob/6f1db35d8abb01538ac55e92272fa3a85a8e5910/lncrawl/core/scraper.py#L104-L105